CONTRIBUTING.md

```diff
@@ -62,12 +62,32 @@ Many linting issues can be automatically fixed by running `yarn fix`.
 ## Local docker setup
 This bot comes shipped with possibility to run it in docker.
 
-ts-node-esm  ./node_modules/.bin/knex migrate:make create_tables -x ts
```

Comment:
Why is that removed?

Reply:
It's removed because it is now handled by the `npm run db-migrate-latest` command.